src/Frame.js

```diff
@@ -0,0 +1,60 @@
+function Frame() {
+  // This tracks the rolls for a frame
+  this.scores = [null, null];
```

Comment:
I think most of these should be private state, indicated as such with an underscore